configs/machines/juwels.yaml

```diff
@@ -12,7 +12,7 @@ accounting: true
 #compiler_mpi: intel_2019a_psmpi2019
 #iolibraries: system_libs
 # for FOCI/FOCIOIFS use
-compiler_mpi: intel2019_impi2019
+compiler_mpi: intel2022_ompi2022
```

Comment:
Why is this needed here when it's already defined in `awicm3.yaml`? Is it because one of the bugs we were talking about right?

Reply:
I recall now. I did this because when you load 2019 you get a message that it will be deleted soon:
  **This stage is deprecated and will be removed soon. Please consider moving to a new stage (2024 or newer)**